Makefile

```diff
@@ -143,3 +143,13 @@ stop-elasticsearch:
 
 wait-elasticsearch:
 	$(call wait-for, localhost:9200)
+
+run-standalone: build
+	podman run --rm \
+	           --env QUERIDO_DIARIO_ELASTICSEARCH_INDEX=$(REMOTE_ELASTICSEARCH_INDEX) \
+	           --env QUERIDO_DIARIO_ELASTICSEARCH_HOST=$(REMOTE_ELASTICSEARCH_HOST) \
+	           --env QUERIDO_DIARIO_DATABASE_CSV=$(QUERIDO_DIARIO_DATABASE_CSV) \
+	           --publish $(API_PORT):$(API_PORT) \
+	           --user=$(UID):$(UID) \
+	           $(IMAGE_NAMESPACE)/$(IMAGE_NAME):$(IMAGE_TAG) \
+	           python main
```

Comment:
Is this working in your env? I'm not sure if this will work. With this command, it will run the process inside a container/namespace which is not guarantee to has the Elasticsearch running.

Reply:
the idea is to use with another Elasticsearch, e.g., when you setup with real data using [data processing](https://github.com/okfn-brasil/querido-diario-data-processing/pull/22) to create a real scenery.

The problem here is do a `make run` and try to create another Elasticsearch when you are running Elasticsearch with `querido-diario-data-processing`. Because of that, you need to run with 2 parameters as you can see at [README](https://github.com/andreformento/querido-diario-api/tree/run-api-using-remote-elastic-search#running-with-a-remote-elastic-search):
```shell
REMOTE_ELASTICSEARCH_INDEX="querido-diario" REMOTE_ELASTICSEARCH_HOST="my.ip" make run-standalone
```

I think another option is to create all environment in one place. But I don't know if is a good idea.

What you think about this @jvanz  ?